homework_01/calc.py

```diff
@@ -0,0 +1,16 @@
+"""
+task_01.md
+Warming up
+    1. please create your own git repository on github.
+    2. (optional) setup pre-commit hook with black and isort formatting for the repo
+    3. initialize .gitignore in the repository root (you can use this sample)
+    4. create a homework1 directory in the repo
+    5. then copy the sample_project into the directory.
+
+    6. fix all bugs in the sample_project code
+    7. write an extra test for each found bug
+"""
+
+
+def check_power_of_2(a: int) -> bool:
+    return a and not (bool(a & (a - 1)))
```

Comment:
Please be accurate with `and`  it may return the left of the right operand. In that case, left is `int` and right is `bool`.
It's ok because the left part could be returned only if a == 0 and this is almost the same as False.  But for this task let's fix it. 

Also, you can omit parenthesis after `not`.